transcom/mymove
# B-18327 Remove fields -"Requested delivery date", "Scheduled delivery date",  "Actual delivery date", and "Required Delivery Date"

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a874041)
B-18327

## Summary

AC copied from the Agility Ticket: Given that I am a TOO, when reviewing the PPM dashboard, then I will not see fields "requested delivery date", "Scheduled delivery date",  "Actual delivery date", and "Required Delivery Date"

It is important to note that the document containing screenshots which is attached to the Agility Ticket (PPM TOO PPM Dashboard issues.docx) shows issues also with the pick-up dates section.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Open a PPM Move as a TOO user and view the MTO tab.
2. Observe that the html  tables which previously showed the following values is no longer visible. 
    -Required Deliver Date
    -Requested Pickup Date
    -Scheduled Pickup Date
    -Actual Pickup Date
    -Requested Delivery Date
    -Scheduled Delivery Date
    -Actual Delivery Date
3. Since this should be conditional on wether or not it's a PPM, open a non-PPM move as TOO user and view the MTO tab.
4. Observe that the html tables which previously showed the above values are now visible.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-01-16 at 2 36 42 PM](https://github.com/transcom/mymove/assets/147739091/89533ef5-409b-4c00-9352-a14ae9502ecf)
